google/cloud/storage/examples/storage_bucket_samples.cc

```diff
@@ -615,7 +615,7 @@ void GetBucketLifecycleManagement(google::cloud::storage::Client client,
     throw Usage{"get-bucket-lifecycle-management <bucket-name>"};
   }
   auto bucket_name = ConsumeArg(argc, argv);
-  //! [view_lifecycle_management] [START storage_view_lifecycle_management]
+  // [START storage_view_lifecycle_management_configuration]
```

Comment:
I don't yet grok when we do/don't need each variety of tag, if substrings matter, etc.

Reply:
I sent a doc about this earlier this evening, happy to answer specific questions there.

Reply:
Thanks.